addons/crm_claim/data/crm_claim_data.xml

```diff
@@ -1,63 +1,56 @@
-<?xml version="1.0"?>
-<openerp>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
     <data noupdate="1">
 
-        <!--
-        Claims Categories
-        -->
+        <!--Claims Categories-->
         
-        <record model="crm.claim.category" id="categ_claim1">
+        <record id="categ_claim1" model="crm.claim.category">
             <field name="name">Factual Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
             </record>
         
-        <record model="crm.claim.category" id="categ_claim2">
+        <record id="categ_claim2" model="crm.claim.category">
             <field name="name">Value Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
              </record>
         
-        <record model="crm.claim.category" id="categ_claim3">
+        <record id="categ_claim3" model="crm.claim.category">
             <field name="name">Policy Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
         </record>
     
-        <!--
-        Case Category2
-        -->
+        <!--Case Category2-->
         
-        <record model="utm.campaign" id="claim_source1">
+        <record id="claim_source1" model="utm.campaign">
             <field name="name">Corrective</field>
         </record>
         
-        <record model="utm.campaign" id="claim_source2">
+        <record id="claim_source2" model="utm.campaign">
             <field name="name">Preventive</field>
         </record>
     
-        <!--
-        Case Stage
-        -->
+        <!--Case Stage-->
         
-        <record model="crm.claim.stage" id="stage_claim1">
+        <record id="stage_claim1" model="crm.claim.stage">
             <field name="name">New</field>
             <field name="sequence">1</field>
             <field name="case_default" eval="True"/>
         </record>
-        <record model="crm.claim.stage" id="stage_claim5">
+        <record id="stage_claim5" model="crm.claim.stage">
             <field name="name">In Progress</field>
             <field name="sequence">27</field>
             <field name="case_default" eval="True"/>
         </record>
-        <record model="crm.claim.stage" id="stage_claim2">
+        <record id="stage_claim2" model="crm.claim.stage">
             <field name="name">Settled</field>
             <field name="sequence">28</field>
             <field name="case_default" eval="True"/>
         </record>
-        <record model="crm.claim.stage" id="stage_claim3">
+        <record id="stage_claim3" model="crm.claim.stage">
```

Comment:
Ohhh please every where found so many unusual diff , Can you please re-check once this code ?
![ohmygod](https://cloud.githubusercontent.com/assets/14940179/12152111/d320f8da-b46f-11e5-9e0a-dd84223e696b.gif)